packages/tools/pie-icons-webc/package.json

```diff
@@ -30,13 +30,11 @@
     "PIE"
   ],
   "scripts": {
-    "build": "yarn clean && yarn build:icons && yarn build:dist",
```

Comment:
We're now making use of dependent tasks via turborepo so that we can benefit from caching.